### DISCUSSION TITLE
Browser within Customtkinter

### DISCUSSION THREAD

**Discussion by RAHULFROST7** [TomSchimansky/CustomTkinter#1423] [Q&A]
import tkinter as tk
import webview

root = tk.Tk()
root.geometry("800x600")

# Create a new web browser window
browser = webview.create_window("My Browser", "https://www.google.com")

# Run the main event loop
webview.start()

root.mainloop()

as you can see i can integrate browser using tkinter but i cant with customtkinter ....what should i do

**Comment by Akascape** [comment]
@RAHULFROST7 Every tkinter related library can be used with customtkinter.
I recommend using [`tkinterweb`](https://pypi.org/project/tkinterweb/) for adding a browser window.

```python
import tkinterweb # pip install tkinterweb
import customtkinter as ctk

def open_website():
    response = ctk.CTkInputDialog(text="Enter website name")
    website = response.get_input()
    web_frame.load_website(website) # load any website
        
root = ctk.CTk()
root.title("CTkBrowser")

frame = ctk.CTkFrame(root)
frame.pack(fill="both", expand=True, padx=30, pady=30)

label = ctk.CTkButton(frame, text="CUSTOMTKINTER BROWSER", command=open_website)
label.pack(fill="both", pady=10, padx=20)

web_frame = tkinterweb.HtmlFrame(frame)
web_frame.load_website("www.google.com") # load the website
web_frame.pack(fill="both", expand=True, padx=10, pady=10)

root.mainloop()
```

**Reply by ByteJoseph** [reply]
Modern websites are not working with tkinterweb

**Reply by Akascape** [reply]
@its-joseph-230606 I recommend not to implement a browser window. Instead use webbrowser module to open any link in the default browser.
